client/components/docs-example/index.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+
+const DocsExample = ( {
```

Comment:
This is just a presentational component, a class is overkill.
I kind of like the idea of [presentational and container components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.b9txfztsu).
However I am ok with refactoring if you guys prefer the `React.createClass` approach.